pynest/nest/lib/hl_api_simulation.py

```diff
@@ -188,6 +188,102 @@ def SetKernelStatus(params):
     params : dict
         Dictionary of parameters to set.
 
+    Other parameters
+    ----------------
+    The following parameters can be written to the kernel status dictionary.
+
+    resolution : double
+        The resolution of the simulation (in ms)
+    time : double
+        The current simulation time
```

Comment:
You should either give units for all quantities or for none.

Reply:
Hey @heplesser thanks for spotting this. I would argue in favor of providing units always. But I am not sure which units are needed in each case. 

Is it always milliseconds for time-related keywords? What about the rest?

Reply:
Good point! In the kernel, all time-related keywords are in ms. The only other quantities with units I spot are potentially the various buffer sizes, although they may also be number of buffer elements, not size in bytes (@suku248 @jakobj could you help on this?); and the wfr_tol, which might be relative or absolute (@janhahne @ddahmen @rgutzen could you provide details?).

Reply:
@heplesser wft_tol is the absolute tolerance for the iterative scheme to stop (every membrane potential/rate change from one iteration to the next needs to be below this value for all neurons that use waveform relaxation).

Reply:
@janhahne So this means that if I simulate a network of spiking neurons with gap junctions, `wfr_tol` would have units of mV, because it refers to membrane potential, and if I simulate a network of rate neurons it would have units of spikes/second?

Reply:
@heplesser yes, exactly :)